packages/contracts/src/current/interfaces/IExchange_v1.sol

```diff
@@ -0,0 +1,226 @@
+pragma solidity ^0.4.19;
+
+contract IExchange_v1 {
```

Comment:
We can leave out the `_v1` for all of the current packages. Also, I would prefer putting the interfaces in their respective folders, rather than creating a new `interfaces` folder (i.e put this in the `Exchange` folder.

Reply:
My intention for this was to have the old interface around. This could be useful if we want to try something where the new Exchange contract calls the existing Exchange contract.

Reply:
So essentially only have interfaces for old versions of the contracts? That makes sense. But why keep the `IExchange_v1` interface around if no one is using it?

Reply:
Good catch! The whole `/interfaces/` folder should not be there. Must have messed up some of the rebasing.

There is an old version of the exchange in `previous/Exchange/Exchange_v1.sol`. I don't think it's likely that we will use it, but there might be cases where we want to access the old exchange from the new exchange.

Simple example where this is necessary: If the new exchange is backwards compatible (same order+sig would be accepted) then we need to check if it has been filled/cancelled by the other exchange. Otherwise an adversary could submit it to both and fill it twice. (note to self: this only solves half of the problem, submitting to new, then old, would still work).